scaffolding-chef-infra/lib/scaffolding.sh

```diff
@@ -10,14 +10,23 @@ fi
 scaffolding_load() {
   : "${scaffold_chef_client:=chef/chef-client}"
   : "${scaffold_chef_dk:=chef/chef-dk}"
+  : "${scaffold_cacerts:=}"
   : "${scaffold_policyfile_path:=$PLAN_CONTEXT/../policyfiles}"
   : "${scaffold_data_bags_path:=$PLAN_CONTEXT/../data_bags}"
 
+if [ -n "${scaffold_cacerts}" ]; then
   pkg_deps=(
     "${pkg_deps[@]}"
     "${scaffold_chef_client}"
-    "core/cacerts"
+    "${scaffold_cacerts}"
   )
+else
+  pkg_deps=(
+    "${pkg_deps[@]}"
+    "${scaffold_chef_client}"
+  )
```

Comment:
Should this branch have `"core/cacerts"`?

To avoid duplicate lists of `pkg_deps` where the cacerts is the only difference, what do you think of adding to an existing array?

```bash
pkg_deps=(
  "${pkg_deps[@]}"
  "${scaffold_chef_client}"
)

if [ -n "${scaffold_cacerts}" ]; then
  pkg_deps+=("${scaffold_cacerts}")
else
  pkg_deps+=("core/cacerts")
fi
```

Reply:
Fantastic! This is a portable pattern to the Windows scaffolding (which I'll be working on next!) as well.